packages/suite/src/views/wallet/coinmarket/common/CoinmarketUtils/CoinmarketUtilsProvider.tsx

```diff
@@ -0,0 +1,60 @@
+import styled from 'styled-components';
+import invityApi from 'src/services/suite/invityAPI';
+import { Translation } from 'src/components/suite';
+import { spacingsPx, typography } from '@trezor/theme';
+
+const IconWrap = styled.div`
+    display: flex;
+    align-items: center;
+`;
+const Icon = styled.img`
+    flex: none;
+    width: 20px;
+    border-radius: 2px;
+`;
+
+const IconText = styled.div`
+    flex: auto;
+    width: 100%;
+    align-items: center;
+    margin-left: ${spacingsPx.xs};
+    ${typography.body}
+    color: ${({ theme }) => theme.textDefault};
+`;
+
+interface CoinmarketUtilsProviderProps {
+    exchange?: string;
+    className?: string;
+    providers?: {
+        [name: string]: {
+            logo: string;
+            companyName: string;
+            brandName?: string;
+        };
+    };
+}
+
+export const CoinmarketUtilsProvider = ({
+    exchange,
+    providers,
+    className,
+}: CoinmarketUtilsProviderProps) => {
+    const provider = providers && exchange ? providers[exchange] : null;
+
+    return (
+        <IconWrap className={className}>
+            {provider ? (
+                <>
+                    {provider.logo && (
+                        <Icon
+                            src={`${invityApi.getApiServerUrl()}/images/exchange/${provider.logo}`}
```

Comment:
please create a function `getProviderLogoUrl` (just like `getCoinLogoUrl`) in `invityAPI.ts` and use it in both places